omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -14,7 +14,7 @@ OMERO.server Mac OS X installation walk-through with Homebrew
     the downloads page or build from source.**
```

Comment:
Do not understand why this sentence is here. Does it want to say: ``If you do not want to use homebrew, or if you do not want to build the OMERO.server from source, you can go to the default instructions for UNIX platforms and install the prerequisites and ....????``
Note: It is very confusing that 
  - the sentence says ``all you need`` and then sends you to a page which has much more complicated workflow ?
  - the sentence says, if I paraphrase it ``go to default install page and let yourself be guided to install the prerequisites with Homebrew`` but there is no mention of Homebrew in the default Unix install ?